src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/BandwidthV2Spec.groovy

```diff
@@ -295,6 +295,13 @@ class BandwidthV2Spec extends HealthCheckSpecification {
         flowPathAfterUpdate == flowPath
         checkBandwidth(flowPathAfterUpdate, linksBeforeFlowCreate, linksAfterFlowUpdate)
 
+        and: "System retries to reroute the flow with ignored bandwidth"
```

Comment:
I don't understand this. Test is about update, not reroute. Moreover, it is questionable whether this retry even has to happen at all